### ISSUE TITLE
Usage of deprecated properties in gradle 6

### ISSUE THREAD

**Issue by jcornaz** [trustin/sphinx-gradle-plugin#15]
Hi, thanks for this plugin :-)

I just want to report that there are usages of deprecated gradle API.

It'd be nice to migrate these usages ;-) 

```
> Task :sphinx UP-TO-DATE
Property 'force' has redundant getters: 'getForce()' and 'isForce()'. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'skip' has redundant getters: 'getSkip()' and 'isSkip()'. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'traceback' has redundant getters: 'getTraceback()' and 'isTraceback()'. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'verbose' has redundant getters: 'getVerbose()' and 'isVerbose()'. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'warningsAsErrors' has redundant getters: 'getWarningsAsErrors()' and 'isWarningsAsErrors()'. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'binaryCacheDir' has @Input annotation used on property of type 'File'. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'force' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'skip' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'traceback' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'verbose' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'warningsAsErrors' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
```

**Comment by trustin** [comment]
Thanks a lot for the suggestion. Let me take a look. Please feel free to send a pull request if you already have a fix or you are interested in contribution!

**Comment by jakubhava** [comment]
Having those fixed would be great :) Together with the latest sphinx we would get rid of all the warnings during documentation build.

Thanks for great plugin!

**Comment by trustin** [comment]
Fixed in 2.7.2
